xml/System/IntPtr.xml

```diff
@@ -51,7 +51,19 @@
     <BaseTypeName>System.ValueType</BaseTypeName>
   </Base>
   <Interfaces>
-    <Interface>
+    <Interface FrameworkAlternate="net-5.0">
+      <InterfaceName>System.IComparable</InterfaceName>
```

Comment:
New interfaces are showing up here.

Reply:
I'll assume this is just mdoc being able to differentiate which interfaces are implemented between different versions. The published doc is here - https://docs.microsoft.com/en-us/dotnet/api/system.intptr?view=net-5.0.